client_end_user/shared/src/commonMain/kotlin/presentation/auth/signup/registration/RegistrationUIState.kt

```diff
@@ -2,9 +2,11 @@ package presentation.auth.signup.registration
 
 data class RegistrationUIState(
     val username: String = "",
+    val email: String = "",
     val password: String = "",
     val isLoading: Boolean = false,
     val isUsernameError: Boolean = false,
+    val isEmailError: Boolean = false,
     val isPasswordError: Boolean = false,
```

Comment:
could be seperated in another data class like RegisterValidation or ValidationErrorState or could make a data class for Textfield and that textfield has type and error state...etc